pkg/devfile/parser/data/v2/common/component_helper_test.go

```diff
@@ -150,6 +150,16 @@ func TestGetComponentType(t *testing.T) {
 			},
 			componentType: v1.PluginComponentType,
 		},
+		{
+			name: "Image component",
+			component: v1.Component{
+				Name: "name",
+				ComponentUnion: v1.ComponentUnion{
+					Image: &v1.ImageComponent{},
+				},
+			},
+			componentType: v1.ImageComponentType,
+		},
```

Comment:
Similar to the comment above, do we need to add  a TestIsImage() test for this new component?